assets/airports/egll.OLD.json

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
Why are there 2 files?

Reply:
egll.OLD.json was the original file. I decided to keep it as reference if I needed it. Forgot to delete